frontend/cypress/integration/examples/core/sidebar.spec.ts

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
Does it belong to this PR?

Reply:
Yep, theres an example in `core/sidebar` but no spec file to match it and it's breaking the E2E script.